plugins/rapture/nexus-rapture-plugin/src/main/resources/org/sonatype/nexus/rapture/internal/index.vm

```diff
@@ -20,9 +20,6 @@
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
   <meta name="description" content="Sonatype Nexus"/>
 
-  ## force browser to behave, IE=edge seems to have issues on IE 11 setting stylesheets (maybe other shit too)
-  <meta http-equiv="X-UA-Compatible" content="IE=9,chrome=1"/>
```

Comment:
Extjs5 docs suggest:

    <meta http-equiv="X-UA-Compatible" content="IE=edge">

Why don't we use that vs. removing it?